### ISSUE TITLE
Problem about Lora TIme-moe 50M prediction errors worse than zero shot

### ISSUE THREAD

**Issue by iganggang** [Time-MoE/Time-MoE#92]
Dear Author,
 I tried to lora time-moe 50M model by lora all linear layers, although i tried to revise the parameters, train loss is ok, but its evaluation results are still worse. Other people also has the problem.
here are the results. Could u please give some future direction and suggestions or reasons?
Thank you very much.
<img width="999" height="730" alt="Image" src="https://github.com/user-attachments/assets/9b85f4af-dced-4561-9c69-52edf55a4dd7" />
here is the lora code,

```
        # Lora Model
        lora_config = LoraConfig(task_type="CAUSAL_LM", target_modules='all-linear', inference_mode=False,
                                 # r=32,
                                 # lora_alpha=64,
                                 r=train_config['lora_rank'],
                                 lora_alpha=train_config['lora_alpha'],
                                 lora_dropout=0.1,
                                 )

        config = AutoConfig.from_pretrained('Maple728/TimeMoE-50M', device_map="cuda", trust_remote_code=True)
        # pprint(config)

        original_model = AutoModelForCausalLM.from_config(config, trust_remote_code=True)
        # pprint(original_model)

        lora_model = get_peft_model(original_model, lora_config)
        # pprint(lora_model)

        lora_model.print_trainable_parameters()
        model = lora_model # Set the model to be a LoRA model.
```

**Comment by XUHIT** [comment]
I believe the failure of LoRA may be due to the model's small parameter size. If the authors release a model with a billion-level parameter scale, it would be worth re-running the experiments to verify this.

**Comment by iganggang** [comment]
i try to revise target_modules,
```
         target_modules = [
            "q_proj", "k_proj", "v_proj",  # attention projections
            "o_proj", "gate_proj",  # output and gating
            "up_proj", "down_proj"  # MLP
        ]
```
but the results is worse than zero shot. is it normal?

**Comment by Maple728** [comment]
Thank you for your interest in our work!

I haven’t trained Time-MoE using the LoRA method yet. However, with full fine-tuning, Time-MoE performs better than the zero-shot. In my view, a LoRA-tuned Time-MoE should also outperform zero-shot across model sizes. If the training losses for LoRA and full fine-tuning are comparable, the gap you’re seeing might be due to an implementation mismatch between training and inference.

When I have some free time, I’ll try to implement LoRA training for Time-MoE.

Please feel free to contact us if you have any questions.

**Comment by iganggang** [comment]
i try several times, and am sure to merge the adaptor weights into base models, the results is still worse than zero shot at same training epochs. the training loss during fine-tune or lora is about equals to 0.03. but in the inference phrase, the results are different. 
at my new experiment, results show as below:

<img width="695" height="131" alt="Image" src="https://github.com/user-attachments/assets/77360cd0-bff6-49ce-a2df-91a6cb2246f1" />

**Comment by iganggang** [comment]
I modified many aspects of the target modules, so the results are worse.
